src/providers/token-lists.provider.ts

```diff
@@ -50,7 +52,7 @@ export default {
 
     let allTokenLists = {};
     try {
-      allTokenLists = require<TokenListMap>('/public/data/tokenlists.json');
+      allTokenLists = require<TokenListMap>(`/public/data/tokenlists/tokens-${NETWORK_ID}.json`);
```

Comment:
This will be why some token icons are not displaying. `allTokenLists` now needs to be a reactive property that updates when `networkId.value` updates.

Reply:
The change in this line should fix the token icon problem. Does it still persist?